cmd/rollouts-controller/main.go

```diff
@@ -97,7 +97,7 @@ func newCommand() *cobra.Command {
 			}
 			logutil.SetKLogLogger(logger)
 			logutil.SetKLogLevel(klogLevel)
-			log.WithField("version", version.GetVersion()).Info("Argo Rollouts starting")
+			log.WithField("version", version.GetVersion()).Info("Argo Rollouts starting - Adevinta Version")
```

Comment:
I would envision that the adevinta specifics is handled inside `version` rather than at startup

Reply:
I'll remove this extra string.

Reply:
Done.